documentation/src/main/jbake/content/documentation.adoc

```diff
@@ -1078,10 +1088,44 @@ For beam case, you need to rely on beam pipeline definition and use `component-r
 `org.talend.sdk.component.runtime.beam.TalendIO` provides a way to convert a partition mapper or a processor to an input or processor
 using the `read` or `write` methods.
 
+[source,java]
+----
+public class Main {
+    public static void main(final String[] args) {
+        final ComponentManager manager = ComponentManager.instance()
+        Pipeline pipeline = Pipeline.create();
+        //Create beam input from mapper and apply input to pipeline
+        pipeline.apply(TalendIO.read(manager.findMapper(manager.findMapper("sample", "reader", 1, new HashMap<String, String>() {{
+                    put("file", "/tmp/input.csv");
```

Comment:
we shouldn't hardcode this kind of path because it doesn't work this way in the big data env, it is more a pattern so maybe rename "file", "fileprefix" and remove the .csv from the value?